webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -369,7 +369,7 @@ def mocked_product_versions(**params):
                     'throttle': '99.00',
                     'end_date': end_date,
                     'start_date': '2012-03-08',
-                    'is_featured': True,
+                    'is_featured': False,
```

Comment:
I changed both of these ^ first to see if that broke any tests. It didn't. So I reused this piece of fixture for my new changes.